fineract-avro-schemas/src/main/avro/document/v1/DocumentDataV1.avsc

```diff
@@ -0,0 +1,88 @@
+{
+    "name": "DocumentDataV1",
+    "namespace": "org.apache.fineract.avro.document.v1",
+    "doc": "Metadata emitted by DocumentCreatedBusinessEvent when a file is stored in Fineract.",
+    "type": "record",
+    "fields": [
+        {
+            "default": null,
+            "name": "id",
+            "type": [
+                "null",
+                "long"
+            ]
+        },
+        {
+            "default": null,
+            "name": "parentEntityType",
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        {
+            "default": null,
+            "name": "parentEntityId",
+            "type": [
+                "null",
+                "long"
+            ]
+        },
+        {
+            "default": null,
+            "name": "name",
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        {
+            "default": null,
+            "name": "fileName",
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        {
+            "default": null,
+            "name": "size",
+            "type": [
+                "null",
+                "long"
+            ]
+        },
+        {
+            "default": null,
+            "name": "type",
+            "type": [
+                "null",
+                "string"
+            ]
+        },
+        {
+            "default": null,
+            "name": "location",
```

Comment:
I dont think we should disclose this information...

Reply:
@adamsaghy, Will remove this. I don't think we will need the location to access the document through API calls.